src/main/java/io/vertx/ext/sql/SQLClient.java

```diff
@@ -42,11 +44,183 @@
    * Close the client and release all resources.
    * Call the handler when close is complete.
    *
-   * @param completionHandler the handler that will be called when close is complete
+   * @param handler the handler that will be called when close is complete
    */
-  void close(Handler<AsyncResult<Void>> completionHandler);
+  void close(Handler<AsyncResult<Void>> handler);
+
   /**
    * Close the client
    */
   void close();
+
+  /**
+   * Execute a one shot SQL statement. This method will reduce the boilerplate code by getting a connection from the
```

Comment:
rather:

Execute a single SQL statement, this method acquires a connection from the the pool and (this project), executes the SQL statement and returns it back after the execution.